app/controllers/admin/rewards_controller.rb

```diff
@@ -36,8 +36,12 @@ def update
     end
 
     def destroy
-      reward.destroy
-      redirect_to admin_rewards_url, notice: 'Reward was successfully destroyed.'
+      if reward.destroy
+        notice = 'Reward was successfully destroyed.'
+      else
+        notice = reward.errors.full_messages[0]
```

Comment:
this is a little bit uncommon to use full messages in a notice

Reply:
Fixed. Thank you for the feedback.